src/lib/hooks/swap/useSyncTokenAmounts.ts

```diff
@@ -24,23 +32,16 @@ function useDefaultToken(
   return token
 }
 
-interface UseSwapDefaultsArgs {
-  defaultInputAddress?: DefaultAddress
-  defaultInputAmount?: string
-  defaultOutputAddress?: DefaultAddress
-  defaultOutputAmount?: string
-}
-
-export default function useSyncSwapDefaults({
-  defaultInputAddress,
+export default function useSyncTokenAmounts({
+  defaultInputTokenAddress,
```

Comment:
I think this hook should retain at least the `defaults` portion

maybe `useSyncTokenDefaults`?

also, note that the `Token` entities are being synced here as well, not just the amounts.

Reply:
I like `useSyncTokenDefaults`, will change.